src/wallet/wallet.h

```diff
@@ -71,6 +71,8 @@ static const CAmount DEFAULT_FALLBACK_FEE = 0;
 static const CAmount DEFAULT_DISCARD_FEE = 10000;
 //! -mintxfee default
 static const CAmount DEFAULT_TRANSACTION_MINFEE = 1000;
+//! -consolidatefee default
+static const CAmount DEFAULT_CONSOLIDATE_FEERATE = 10000; // 10 sat/vbyte
```

Comment:
definitely not going to nitpick this but why 10? :)

Reply:
It seemed like a reasonable number

Reply:
Given that we have gotten that question twice again in the last 24h, it would probably be best to add context for that number in the commit messages or code somewhere.

Reply:
I added to the commit message that it was chosen arbitrarily.